src/layout/Group/SummaryGroupComponent.tsx

```diff
@@ -79,7 +79,7 @@ export function SummaryGroupComponent({
             <DisplayGroupContainer
               key={`summary-${targetNode.item.id}-${idx}`}
               id={`summary-${targetNode.item.id}-${idx}`}
-              groupNode={targetNode}
+              groupNode={targetNode as LayoutNode<HNonRepGroup, 'Group'>}
```

Comment:
Same here - better to wrap in `node.isNonRepGroup()` than to cast. Also, why the need to check if `targetNode` is a non-repeating group below here if you've already told the type system it always is? 🤔

Reply:
I guess I moved a little fast with these 😅